### ISSUE TITLE
ctypes for linux

### ISSUE THREAD

**Issue by FuckingToasters** [Aran404/Raptor-Multi-Tool#5]
Hey, i know the Title is kinda clickbait BUT you wrote in the readme, that ctypes only works on windows and linux users should remove it. 
It seems like you dont know what the `os` libary can do. You can check if the system is windows or linux and only if its windows run the ctype stuff. 
`if os.name == "nt": # if the system is windows`
If you need help from a real coder just hit me up. Im not skidding like you do.

**Comment by DaRkSurface** [comment]
Same problem for me 

Do: ```if os.name == "nt":  "do something idk" else:```

**Comment by Aran404** [comment]
Imagine being 21 and wasting your life trash talking get a hold of yourself, i couldn't care less about ctypes you the one that is telling people to pip install os/sys, goofy

**Comment by FuckingToasters** [comment]
I not told that they need to install it i just said how to fix it for the Linux thing instead of just removing all the lines that are related to ctypes.

**Comment by FuckingToasters** [comment]
> > Imagine being 21 and wasting your life trash talking get a hold of yourself, i couldn't care less about ctypes you the one that is telling people to pip install os/sys, goofy
> 
> isint os with python, you dont have to install it

Yes indeed it is. I just added it to be sure but because it not break it, the installation ends and it continue with the next one there is no reason to remove it from my repo. Users still can remove it tho but it's unnecessary work and I'm lazy as fuck.
